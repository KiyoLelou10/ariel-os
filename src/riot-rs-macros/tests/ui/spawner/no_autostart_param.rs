#![no_main]
#![feature(used_with_arg)]

// As the macro will fail, this import will not get used
#[allow(unused_imports)]
use riot_rs::asynch::Spawner;

// FAIL: the macro expects a mandatory `autostart` parameter
#[riot_rs::spawner]
fn main(spawner: Spawner) {}
