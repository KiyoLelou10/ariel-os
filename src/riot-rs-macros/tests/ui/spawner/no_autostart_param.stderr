error: custom attribute panicked
 --> tests/ui/spawner/no_autostart_param.rs:9:1
  |
9 | #[riot_rs::spawner]
  | ^^^^^^^^^^^^^^^^^^^
  |
  = help: message: the `autostart` parameter must be provided
